DEVELOPING.md

```diff
@@ -18,10 +18,9 @@ This extension is built using the [gradle-intellij-plugin](https://github.com/Je
 
 1. Clone this repository: `git clone git@github.com:OpenLiberty/liberty-tools-intellij.git`
 2. Clone the lsp4ij repository: `git clone git@github.com:MicroShed/lsp4ij.git`
-3. Build lsp4ij: `./gradlew jar`
-4. Save it in your local Maven repository: `./gradlew publishToMavenLocal` (this also runs the `jar` task)
-2. Import this repository as a Gradle project in IntelliJ IDEA
-3. Run `./gradlew runIde --stacktrace`. A new IntelliJ IDEA window will launch with the Liberty Tools plugin installed to it. You can connect the IntelliJ IDEA debugger to this process to debug the plugin.
+3. Build lsp4ij and save it in your local Maven repository: `cd` to your local lsp4ij repository and run `./gradlew publishToMavenLocal`. Alternatively, build only the jar: `./gradlew jar`
```

Comment:
> your local lsp4ij repository

Does this mean the same repo you cloned in the previous step?

Reply:
Yes, that is what it is referring to.